### PR TITLE
Brand component

### PR DESCRIPTION
**Pull request by ileenshaar**

### CODE REVIEW

File: src/components/brand-section/brand-section.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  "https://images.unsplash.com/photo-1537996194471-e657df975ab4?auto=format&fit=crop&w=400&h=250",
+  "https://images.unsplash.com/photo-1537944434965-cf4679d1a598?auto=format&fit=crop&w=400&h=250&q=60",
+  "https://images.unsplash.com/photo-1512341689857-198e7e2f3ca8?auto=format&fit=crop&w=400&h=250&q=60",
+  "https://images.unsplash.com/photo-1537996194471-e657df975ab4?auto=format&fit=crop&w=400&h=250",
+  "https://images.unsplash.com/photo-1537944434965-cf4679d1a598?auto=format&fit=crop&w=400&h=250&q=60",
+];
+
+export function BrandSection() {
+  return (
+    <>
```
Comment (mahmoud-alragabi) [inline src/components/brand-section/brand-section.jsx]: remove the fragment, you already are returning a single element no need to wrap it with a fragment

File: src/components/brand-section/brand-section.jsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+          height: 263,
+        }}
+      >
+        <Typography
+          fontSize={"34px"}
+          fontWeight={"500"}
+          sx={{ marginBottom: "20px" }}
+        >
+          Shop by Brands
+        </Typography>
```
Comment (mahmoud-alragabi) [inline src/components/brand-section/brand-section.jsx]: abstract it to its own component it's used in many places

File: src/components/brand-section/brand-section.jsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
+import * as React from "react";
+import { Typography, Paper, Box } from "@mui/material";
+import { ProductImage } from "../product-image/ProductImage";
+
+const brands = [
```
Comment (mahmoud-alragabi) [inline src/components/brand-section/brand-section.jsx]: better renamed as brand images or brand image links, even you got confused and thought it was an object and used its id at the end

File: src/components/brand-section/brand-section.jsx [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+            },
+            "& > :not(style)": {
+              mr: 6,
+              height: 187,
+            },
+          }}
+        >
+          {brands.map((brand) => {
+            return (
+              <Paper key={brand.id} sx={{ minWidth: "168px" }}>
```
Comment (mahmoud-alragabi) [inline src/components/brand-section/brand-section.jsx]: brand is a link doesn't have an id prop

**Review by mahmoud-alragabi — CHANGES_REQUESTED** [review]

**Review by ferassmohana — APPROVED** [review]

**Pull request merged**